docs/diagrams/LogicClassDiagram.puml

```diff
@@ -1,12 +1,13 @@
 @startuml
 !include style.puml
+scale 561*558
 skinparam arrowThickness 1.1
 skinparam arrowColor LOGIC_COLOR_T4
 skinparam classBackgroundColor LOGIC_COLOR
```

Comment:
I noticed changes of color in the generated diagram. Is that intentional?

Reply:
Seems to be due to a difference in software (Different version of PlantUML/Graphviz or maybe Different OS), since the puml file used is the same